ssl/ssl_locl.h

```diff
@@ -551,6 +551,8 @@ struct ssl_session_st {
         /* Session lifetime hint in seconds */
         unsigned long tick_lifetime_hint;
         uint32_t tick_age_add;
+        unsigned char *tick_nonce;
+        size_t tick_nonce_len;
```

Comment:
Right, so, I'm wondering if I should change PR #3802 away from using ASN1_OCTET_STRING in the SSL_SESSION.